lib/ansiblelint/rules/PackageIsNotLatestRule.py

```diff
@@ -29,5 +29,10 @@ class PackageIsNotLatestRule(AnsibleLintRule):
     tags = ['repeatability']
 
     def matchtask(self, file, task):
-        return (task['action']['__ansible_module__'] in ['yum', 'apt', 'package'] and
-                task['action'].get('state') == 'latest')
+        return (task['action']['__ansible_module__'] in ['yum', 'apt', 'dnf', 'homebrew', 'pacman',
```

Comment:
for readability, I'd define a list of package manager modules in a variable and then do the test against that list. You'll have far fewer line breaks.